resource-managers/kubernetes/architecture-docs/submission-client.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: global
+title: Implementation of Submitting Applications to Kubernetes
+---
+
+
+Similarly to YARN and Standalone mode, it is common for Spark applications to be deployed on Kubernetes through the
```

Comment:
On the whole this document seems verbose - if we could use bulleted lists or diagrams in place of some of these discourses that would improve readability, but I'm not sure how to best represent the information in that way.